src/Drivers/IpoptAdapter_driver_ex9.cpp

```diff
@@ -0,0 +1,103 @@
+#include "IpIpoptApplication.hpp"
+
+#include "IpoptAdapter.hpp"
+//use HiOp's Ex5 - sparse NLP
+#include "nlpSparse_ex9.hpp"
+
+#include "nlpSparse_ex6.hpp"
+#include "hiopNlpFormulation.hpp"
+#include "hiopAlgFilterIPM.hpp"
+
+#include <iostream>
+
+using namespace Ipopt;
+using namespace hiop;
+// Example of how to use IpoptAdapter to solve HiOP-specified problems with Ipopt
+
+int main(int argv, char** argc)
+{
+  //instantiate a HiOp problem
+  int nx = 1000;
+  int S = 1920;
+  int nS = 5; 
+  double x0[nx]; 
+  for(int i=0;i<nx;i++) x0[i] = 1.0;
+    
+  Ex6 nlp_interface(nx, 1.0);
+  hiopNlpSparse nlp(nlp_interface);
+  nlp.options->SetStringValue("Hessian", "analytical_exact");
+  nlp.options->SetStringValue("duals_update_type", "linear"); 
+//  nlp.options->SetStringValue("duals_init", "zero"); // "lsq" or "zero"
+  nlp.options->SetStringValue("compute_mode", "cpu");
+  nlp.options->SetStringValue("KKTLinsys", "xdycyd");
+  // nlp.options->SetStringValue("KKTLinsys", "full");
+  // nlp.options->SetStringValue("write_kkt", "yes");
+  // nlp.options->SetIntegerValue("max_iter", 100);
+  nlp.options->SetNumericValue("mu0", 0.1);
+  hiopAlgFilterIPMNewton solver(&nlp);
```

Comment:
I would remove commented lines and document each option. For example
```suggestion
  nlp.options->SetStringValue("duals_update_type", "linear"); 
  nlp.options->SetStringValue("duals_init", "zero");   // "lsq" or "zero"
  nlp.options->SetStringValue("compute_mode", "cpu");  // cpu, hybrid or gpu
  nlp.options->SetStringValue("KKTLinsys", "xdycyd");  // xdycyd or full
  nlp.options->SetStringValue("write_kkt", "no");      // yes/no
  nlp.options->SetIntegerValue("max_iter", 100);
  nlp.options->SetNumericValue("mu0", 0.1);
  hiopAlgFilterIPMNewton solver(&nlp);
```